src/vm/fieldmarshaler.h

```diff
@@ -116,6 +116,8 @@ BOOL HasLayoutMetadata(Assembly* pAssembly, IMDInternalImport *pInternalImport,
 //=======================================================================
 BOOL IsStructMarshalable(TypeHandle th);
 
+BOOL IsFixedBuffer(mdFieldDef field, IMDInternalImport *pInternalImport);
```

Comment:
Please remove this from the header. This doesn't need to be exported from the compilation unit.